src/pages/favorites/favorites.tsx

```diff
@@ -31,7 +39,7 @@ function Favorites({ favoriteList }: FavoritesProps): JSX.Element {
             })}
           >
             {isNotEmpty ? (
-              <FavoriteList favoriteList={favoriteList} />
+              <FavoriteList favoriteList={favorites} />
```

Comment:
Если компонент используеся только с данными из selectFavorites

```suggestion
              <FavoriteList />
```